### DISCUSSION TITLE
Can't display rack view

### DISCUSSION THREAD

**Discussion by sweetnight19** [netbox-community/netbox#9528] [Q&A] [answered]
I can't see the rack view, download from svg it works
 
![Screenshot_1](https://user-images.githubusercontent.com/24542429/173568118-ddf6754d-b667-4d31-819b-8a17388ce7e5.jpg)

**Comment by jeremystretch** [comment, marked as answer]
[This](https://github.com/netbox-community/netbox/wiki/Common-Issues#rack-elevations-do-not-display-v27) may be your issue.

**Reply by sweetnight19** [reply]
i add on virtualhost:

···
Header always unset X-Frame-Options
Header set X-Frame-Options "SAMEORIGIN"
···

thanks for your answer, it works now!!! 😀
